### ISSUE TITLE
Musicbrainz Album Artist IDs separator corrupted

### ISSUE THREAD

**Issue by tfranken90** [jellyfin/jellyfin#13002]
### This issue respects the following points:

- [x] This is a **bug**, not a question or a configuration issue; Please visit our forum or chat rooms first to troubleshoot with volunteers, before creating a report. The links can be found [here](https://jellyfin.org/contact/).
- [x] This issue is **not** already reported on [GitHub](https://github.com/jellyfin/jellyfin/issues?q=is%3Aopen+is%3Aissue) _(I've searched it)_.
- [x] I'm using an up to date version of Jellyfin Server stable, unstable or master; We generally do not support previous older versions. If possible, please update to the latest version before opening an issue.
- [x] I agree to follow Jellyfin's [Code of Conduct](https://jellyfin.org/docs/general/community-standards.html#code-of-conduct).
- [x] This report addresses only a single issue; If you encounter multiple issues, kindly create separate reports for each one.

### Description of the bug

After upgrading to 10.10 the Musicbrainz Album Artist Id separator appears corrupted when refreshing metadata. I thought it might be an issue with newly added files, but it happens when I refresh audio already in my library. 

### Reproduction steps

1. Update Jellyfin
2.  Refresh metadata (replace all metadata)

### What is the current _bug_ behavior?

The separator for multiple musicbrainz ID appears corrupted. 

### What is the expected _correct_ behavior?

These separators should be semicolons. 

### Jellyfin Server version

10.10.0+

### Specify the build version

10.10.0

### Environment

```markdown
- OS: Fedora Atomic 41
- Linux Kernel: 6.11.5
- Virtualization: Podman (ghcr.io/jellyfin/jellyfin:latest)
- Clients: Browser
- Browser: Firefox
```

### Jellyfin logs

```shell
uploaded as text file
```

### FFmpeg logs

```shell

```

### Relevant screenshots or videos

![Image](https://github.com/user-attachments/assets/6f20b0fc-dd1b-445d-91c7-ea06d7de525c)

### Additional information

[jellyfin.log](https://github.com/user-attachments/files/17689501/jellyfin.log)

**Comment by felix920506** [comment]
Are the files tagged using musicbrainz Picard?

**Comment by tfranken90** [comment]
Yes.  They are flac files. I also run this script in Picard to set multiartists:

$set(_artists,%artists%)
$setmulti(artist,%_artists%)
$setmulti(artistsort,%_artists_sort%)
$setmulti(albumartist,%_albumartists%)
$setmulti(albumartistsort,%_albumartists_sort%)

**Comment by crobibero** [comment]
Might already be fixed by https://github.com/jellyfin/jellyfin/pull/12962

**Comment by gnattu** [comment]
That requires the enablement of "use custom delimiters" though. This looks like Picard is separating tag values with null char even on FLAC files, and if that is the case the mentioned PR will fix it if the "use custom delimiters" option is enabled.

**Comment by tfranken90** [comment]
@gnattu With the changes to multi-artist tagging in 10.10, what is the best options for settings in picard? I currently use ID3v2 version 2.3, text encoding UTF-16,  and join multiple using / I also use the tagging script mentioned above. I'd rather use the simplest settings that offer the greatest compatibility with jellyfin, is there a recommended combination of settings for that?

**Comment by gnattu** [comment]
Oh wait, I found the problem.

Those tags technically supports multiple values but before 10.10 we only store the first one, but in 10.10 we are trying to parse all at once but not really separating them, and it seems like Picard does not separate them by null char as default but uses `/`.

I'm now a bit confused on how to implement this now, because those fields are very picard specific extensions

**Comment by gnattu** [comment]
> [@gnattu](https://github.com/gnattu) With the changes to multi-artist tagging in 10.10, what is the best options for settings in picard? I currently use ID3v2 version 2.3, text encoding UTF-16, and join multiple using / I also use the tagging script mentioned above. I'd rather use the simplest settings that offer the greatest compatibility with jellyfin, is there a recommended combination of settings for that?

Use ID3v2.4 with UTF-8

**Comment by gnattu** [comment]
It turns out we have to revert to 10.9 and before behavior as out metadata providers do not handle multiple IDs. I made a PR for this.
